### ISSUE TITLE
Bug with Racial Shift

### ISSUE THREAD

**Issue by Dexac3** [gamefreedomgit/Frostmourne#1679]
Every time when use npc for Racial shift i change from Orc to Human .
When i close wow and log again Racial are change to Orc again.
Try with another character dont have problem's

**Comment by danniellos94** [comment]
Are you on PTR Server ? Because it works like this on PTR but on Live server works for me with no issues.

**Comment by Dexac3** [comment]
No im not on PTR server.
I have 2 other character and dont have problems with racial shift.
Only with orc dk i just skip the dk zone with NPC

**Comment by UltraNix** [comment]
@Dexac3 Still valid?

**Comment by Dexac3** [comment]
Yes,  someone admin fix it for a little but orc racial is back again

**Comment by GM-Aule** [comment]
I have another case:
- Blood elf DK with orc racial
- Resets every relog
- Character was created less than a month ago
- Skipped SK quest chain through NPC
- Other characters on the same account or other accounts of the same player don't seem to have this issue.

**Comment by Riztazz** [comment]
fixed
